### DISCUSSION TITLE
Sandrine @ Method (UK) (she/her)

### DISCUSSION THREAD

**Discussion by SandrineResponsibleDesigner** [Green-Software-Foundation/discussions#85] [Introductions]
Hi!
I'm Sandrine, designer specialising in Strategy, Innovation and Experience Design (incl. UX/UI design). 
I initiated the Sustainability practice at [Method](https://www.method.com/), which consists in up-skilling our team with Responsible Design, Systems Thinking and Green Software Engineering techniques to create and launch solutions that keep creating value for our clients and their users, whilst not drawing too much on our environmental resources.
Method belongs to [GlobalLogic](https://www.globallogic.com/) and [Hitachi](https://www.hitachi.com/) and we're excited to be part of a global group that truly cares about the environmental transition of our society, and to build the digital platforms that support it.
We've created the **[Responsible Design Cards](https://docs.google.com/presentation/d/1PjKtTrIQjppasdPP5jz2O6X-eFO-CrQFqCwruMnXhdg/edit#slide=id.g25a9238af31_0_1833)** (tactics for everyday) 
GSF is a go-to reference for our team members. 

I'd like to be part of the GSF community, connect with like-minded creatives, exchange learnings, and possibly work together! 
I'm also looking for inspirational speakers for internal and external events :)

You can find me here: 💬 [https://www.linkedin.com/in/sandrineherbert/](https://www.linkedin.com/in/sandrineherbert/)

**Comment by pdematos12** [comment]
Hi @SandrineResponsibleDesigner 

You may want to join this newly forming project around design.
Sign up details here: https://github.com/Green-Software-Foundation/patterns/discussions/309
